distro/BUILD

```diff
@@ -39,6 +39,7 @@ pkg_tar(
         "//:standard_package",
         "//pkg:standard_package",
         "//pkg/private:standard_package",
+        "//pkg/private/deb:standard_package",
```

Comment:
What about inverting the structure a bit: `//pkg/deb/private`?  Would reduce the complexity of CODEOWNERS somewhat.

This is fine as-is though; just wondering your thoughts.

Reply:
Too much in my head to think about that right now. That is time for another day.

Reply:
Regardless of our current decision, given that this is an implementation detail, we can decide this later if we find it doesn't scale well.